Assets/MRTK/Examples/Demos/EyeTracking/Scenes/EyeTrackingDemo-00-RootScene.unity

```diff
@@ -222,7 +222,7 @@ Transform:
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 256863736}
   m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
-  m_LocalPosition: {x: 0, y: 0, z: 0}
+  m_LocalPosition: {x: 2, y: 0, z: 0}
```

Comment:
What are these `m_LocalPosition` changes moving around? I'm not sure I'm a huge fan of changing the starting position of existing demos arbitrarily, since they're aligned specifically for the camera to start at the origin.

Reply:
That particular change to the local position is for the camera. That is set automatically at startup but immediately overwritten. I can lookup the line of code for you which does that if you like. It has no effect on the demo.

I checked the starting positions for the 3 demos, and with a change of 25cm I couldn't tell they had changed at all. The 2 meter change in the camera above was from when I set the offset large (2m) just so I could be sure it was doing anything.

If it's a big concern, then the same offset can be applied to the Playspace and to the (non-body-locked) assets in the scene, so the experience remains identically the same.

Reply:
And in case it wasn't clear from the meeting yesterday, the changes in local position (of the Playspace) are the entirety of this PR. All other changes are just Unity moving things around between saves, or from a scene having been saved at a previous version of Unity (pre-Unity2018.4.34f1).